aiida/engine/transports.py

```diff
@@ -108,6 +108,10 @@ def do_open():
         try:
             transport_request.count += 1
             yield transport_request.future
+        except asyncio.CancelledError:  # pylint: disable=try-except-raise
```

Comment:
sure, can add it as a debug log (see below), but should not be an error otherwise currently users will see a bunch of misleading errors in the daemon log every time the daemon is stopped